src/controllers/krate/versions.rs

```diff
@@ -109,6 +119,25 @@ fn list_by_date(
             )
         }
         query = query.limit(options.per_page);
+
+        if include.release_tracks {
+            let mut sorted_versions = IndexSet::new();
+            for result in versions::table
+                .filter(versions::crate_id.eq(crate_id))
+                .filter(versions::yanked.ne(true))
```

Comment:
```suggestion
                .filter(versions::yanked.eq(false))
```

😅

Reply:
Oops, I have no idea what I'm thinking.

Or just
```suggestion
                .filter(not(versions::yanked))
```